backend/LexBoxApi/GraphQL/ProjectMutations.cs

```diff
@@ -534,7 +534,15 @@ public async Task<IQueryable<Project>> SoftDeleteProject(
 
         using var transaction = await dbContext.Database.BeginTransactionAsync();
         await dbContext.SaveChangesAsync();
-        await hgService.SoftDeleteRepo(projectCode, timestamp);
+        try
+        {
+            await hgService.SoftDeleteRepo(projectCode, timestamp);
+        }
+        catch (DirectoryNotFoundException)
+        {
+            // If the repo doesn't exist, we don't need to delete it
+            Activity.Current?.AddTag("app.hg.delete", "not-found");
```

Comment:
I'd log the exception as a warning so we can investigate the error if we need to